src/games/common/ScoreBadges.tsx

```diff
@@ -9,26 +9,40 @@ interface IScoreBadgesProps {
   players: IPlayerInRoom[];
   playerID: string;
   colors?: string[];
+  currentPlayer?: string;
 }
 
 export class ScoreBadges extends React.Component<IScoreBadgesProps, {}> {
   render() {
     const badges = this.props.scoreboard.map((score) => {
       const nickname = this.props.players.find((player) => player.playerID.toString() === score.playerID).name;
       const isSelf = score.playerID.toString() === this.props.playerID;
+      const isCurrentPlayer = this.props.currentPlayer === score.playerID;
```

Comment:
This wont work for takesix or other games where you can have multiple people taking actions. We should look at both ctx.activePlayers and, only if activePlayers is null, look into currentPlayer